### DISCUSSION TITLE
Interface and Cross platfrom issues.

### DISCUSSION THREAD

**Discussion by anzal1** [anzal1/Computer-Networks-Mini-Projects#3] [Q&A] [answered]
#1  is completly in java , is it cross-platfrom or generic.

**Comment by Thre4dripper** [comment, marked as answer]
You just have to make the parsing logic same with all other languages and this should work with cross languages, I have tested this with java-kotlin connection.

**Comment by anzal1** [comment]


**Reply by Faizullah9181** [reply]
!!Yes it is Cross Platform
